docs/slides/population_structure/index.qmd

```diff
@@ -501,50 +451,52 @@ pheatmap(mat, cluster_rows = TRUE, cluster_cols = TRUE)
 :::
 ::::
 
-## Allen Ancient DNA Resource (AADR) dataset
 
-<!-- markdownlint-disable MD013 -->
+## Allen Ancient DNA Resource (AADR) dataset
 
 :::: {.columns}
 ::: {.column width="50%"}
 ![](assets/AADR.png){fig-align="center"}
 [https://reich.hms.harvard.edu/allen-ancient-dna-resource-aadr-downloadable-genotypes-present-day-and-ancient-dna-data](https://reich.hms.harvard.edu/allen-ancient-dna-resource-aadr-downloadable-genotypes-present-day-and-ancient-dna-data)
 :::
 
+
 ::: {.column width="50%"}
 ::: {.fragment}
 
 ```{bash}
 #| class-source: my_code_size_abitsmaller
 #| classes: my_code_size_abitsmaller
-head data/DavidReich/AADR.ind
+
+cd /home/nikolay/Documents/Teaching/PopGen_2023/data/DavidReich
```

Comment:
Change absolute hard-coded path to relative. Here, `data/DavidReich` contains excerpts from the ind, snp, and geno files.